dts/bindings/radio_fem/skyworks,sky66407-11.yaml

```diff
@@ -0,0 +1,53 @@
+# Copyright (c) 2023 Nordic Semiconductor ASA
```

Comment:
And change the copyright, please. I'm sorry that I overlooked that before.

Reply:
No, sorry that I overlooked that.